Sources/generate-swift-syntax-builder/Templates/FormatFile.swift

```diff
@@ -225,3 +226,38 @@ private func createBuildableCollectionNodeFormatFunction(node: Node) -> Function
     }
   }
 }
+
+private func createTokenFormatFunction() -> FunctionDecl {
+  let tokenType = SyntaxBuildableType(syntaxKind: "Token")
+  return FunctionDecl(
+    modifiers: [Token.public],
+    identifier: .identifier("_format"),
+    signature: createFormatFunctionSignature(type: tokenType)
+  ) {
+    SwitchStmt(expression: MemberAccessExpr(base: "syntax", name: "tokenKind")) {
+      for token in SYNTAX_TOKENS {
+        SwitchCase(label: SwitchCaseLabel(caseItems: CaseItem(pattern: ExpressionPattern(expression: MemberAccessExpr(name: token.swiftKind))))) {
+          createWithLeadingWithTrailingTriviaCall(token: token)
+        }
+      }
+      SwitchCase(label: SwitchCaseLabel(caseItems: CaseItem(pattern: ExpressionPattern(expression: MemberAccessExpr(name: "eof"))))) {
+        ReturnStmt(expression: "syntax")
+      }
+    }
+  }
+}
+
+func createWithLeadingWithTrailingTriviaCall(token: TokenSpec) -> CodeBlockItem {
```

Comment:
Can we make this private?

```suggestion
private func createWithLeadingWithTrailingTriviaCall(token: TokenSpec) -> CodeBlockItem
```